content/webapp/pages/api/exhibitions-related-content/index.ts

```diff
@@ -21,19 +23,17 @@ export default async (
   const client = createClient({ req });
 
   if (parsedParams.length === 0) {
-    return res.status(200).json(
-      superjson.serialize({
-        exhibitionOfs: [],
-        exhibitionAbouts: [],
-      })
-    );
+    return res.status(200).json({
```

Comment:
should `superjson.serialize` have been replaced with another helper function?

Reply:
Good question!

But I think this is okay – because this is just an empty, default response, there's nothing interesting to change.